docs/usage/openapi/ui_plugins.rst

```diff
@@ -0,0 +1,268 @@
+OpenAPI UI Plugins
+------------------
+
+OpenAPI UI Plugins are designed to allow easy integration with your OpenAPI UI framework of choice. These plugins
+facilitate the creation of interactive, user-friendly API documentation, making it easier for developers and end-users
+to understand and interact with the API.
+
+In addition to serving a default JSON representation of the OpenAPI specification, built-in UI Plugins are available to
+support a range of popular OpenAPI documentation tools, including:
+
+- `Scalar <https://scalar.com/>`_
+- `RapiDoc <https://rapidocweb.com/>`_
+- `ReDoc <https://redocly.com/>`_
+- `Stoplight Elements <https://stoplight.io/open-source/elements>`_
+- `Swagger UI <https://swagger.io/tools/swagger-ui/yy>`_
+- `YAML <https://yaml.org/>`_ (OK, not a UI framework, but configured the same way!)
+
+Each plugin is easily configurable, allowing developers to customize aspects like version, paths, CSS and JavaScript
+resources.
+
+
+Using OpenAPI UI Plugins
+------------------------
+
+Using OpenAPI UI Plugins is as simple as importing the plugin, instantiating it, and adding it to the OpenAPIConfig.
+
+.. tab-set::
+
+    .. tab-item:: scalar
```

Comment:
If we add a tab sync here, when they click e.g., `scalar` on this tab set, it will change in all others inside of the respective document.

Reply:
see https://sphinx-design.readthedocs.io/en/latest/tabs.html#synchronised-tabs

Reply:
got this thanks